raiden/tests/integration/test_integration_events.py

```diff
@@ -525,3 +525,94 @@ def test_secret_revealed(raiden_chain, deposit, settle_timeout, token_addresses)
         app0, deposit - amount, [],
         app1, deposit + amount, [],
     )
+
+
+@pytest.mark.parametrize('number_of_nodes', [2])
+def test_clear_closed_queue(raiden_network, token_addresses, deposit):
```

Comment:
If you're not asserting for the state of the transport layer, i don't see why this should be a integration test.

Reply:
I'm not asserting for the transport, but it depends on it, as the queues are created when the SendMessageEvent's are created (from what I saw). Care to point a better/unit place to put a similar test?